internal/gitrepo/gitrepo.go

```diff
@@ -343,12 +358,13 @@ func (r *Repository) GetCommitsForPathsSinceTag(paths []string, tagName string)
 	return r.GetCommitsForPathsSinceCommit(paths, hash)
 }
 
-// Returns all commits with the given release ID, i.e. where the commit message contains a line of
-// Librarian-Release-Id: <release-id>. These commits are expected to be contiguous, from head,
-// with all commits having a single parent.
-func (r *Repository) GetCommitsForReleaseID(releaseID string) ([]object.Commit, error) {
+// GetCommitsForReleaseID returns all commits with the given release ID, i.e.,
+// where the commit message contains a line of Librarian-Release-Id: <release-id>.
+// These commits are expected to be contiguous, from head, with all commits
+// having a single parent.
+func (r *Repository) GetCommitsForReleaseID(releaseID string) ([]Commit, error) {
 	releaseIDLine := fmt.Sprintf("Librarian-Release-ID: %s", releaseID)
-	commits := []object.Commit{}
+	commits := []Commit{}
```

Comment:
We generally use pointers for this

```
	commits := []*Commit{}
```

Reply:
Done.

Reply:
It looks like this is still `commits := []Commit{}` and needs to be changed